doc/conf.py

```diff
@@ -17,11 +22,9 @@
     "sphinx.ext.intersphinx",
     "numpydoc",
     "nbsphinx",
```

Comment:
@leouieda I tried it locally. It's really nice!!!
Don't forget to remove nbsphinx, since we no longer use notebooks after this PR is merged.

Reply:
Forget it. I just saw your comments in #269.

Reply:
Thanks! I quite like it too.

I'm leaving it because I'll want to have one notebook in there as a "Using PyGMT in Jupyter" tutorial (#269).